scripts/partition_manager.py

```diff
@@ -416,6 +416,29 @@ def first_partition_has_been_aligned(first, solution):
 
 
 def _set_addresses_and_align(reqs, sub_partitions, solution, size, start, dynamic_partitions, dp):
+    # Resolve 'align_next'
+    solution_non_empty = list(filter(lambda part: "EMPTY_" not in part, solution))
+    for i in range(len(solution_non_empty) - 1):
+        if 'placement' in reqs[solution_non_empty[i]] and 'align_next' in reqs[solution_non_empty[i]]['placement']:
+            alignment = reqs[solution_non_empty[i]]['placement']['align_next']
+            if 'placement' not in reqs[solution_non_empty[i + 1]]:
+                reqs[solution_non_empty[i + 1]]['placement'] = dict()
+            if 'align' not in reqs[solution_non_empty[i + 1]]['placement']:
+                reqs[solution_non_empty[i + 1]]['placement']['align'] = dict()
+            else:
```

Comment:
```suggestion
           reqs[solution_non_empty[i+1]].setdefault('placement', {}).setdefault('align', {})
```

This would also remove a layer of indentation from the code following this if-else.